README.md

```diff
@@ -243,6 +243,26 @@ Example:
 
     cancel_transfer($id)
 
+## create\_reversal
+
+Reverses an existing transfer.
+
+Example:
+
+    $ws_stripe->create_reversal(
+        # Transfer ID (required)
+        $xfer_id,
+        {   # POST data (optional)
+            refund_application_fee => 'true'
```

Comment:
It can also take an 'amount' -- if you don't want to reverse the full amount. Not sure if you were wanting to list all options here or not.